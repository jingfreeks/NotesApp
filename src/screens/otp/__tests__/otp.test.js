import React from 'react';
import {render} from '@testing-library/react-native';
import OtpScreen from '../otp';


describe('Otp Screen', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<OtpScreen />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
