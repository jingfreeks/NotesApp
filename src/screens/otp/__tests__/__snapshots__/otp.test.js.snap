// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Otp Screen Should work as expected to get snapshot 1`] = `
<View
  onLayout={[Function]}
  style={
    [
      [
        {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ],
      {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      [
        {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "flexDirection": "row",
            "justifyContent": "space-between",
          },
          {},
        ]
      }
    >
      <TextInput
        autoCorrect={false}
        autoFocus={false}
        cursorColor="#3CB371"
        handleTextChange={[Function]}
        keyboardType="numeric"
        maxLength={1}
        multiline={false}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        selectionColor="#3CB371"
        style={
          [
            {
              "borderBottomWidth": 4,
              "color": "#000000",
              "fontSize": 22,
              "fontWeight": "500",
              "height": 50,
              "margin": 5,
              "textAlign": "center",
              "width": 50,
            },
            {
              "borderRadius": 10,
              "borderWidth": 4,
              "padding": 10,
            },
            {
              "borderColor": "#DCDCDC",
            },
            {
              "borderColor": "#3CB371",
            },
          ]
        }
        testID="otp_input_0"
        value=""
      />
      <TextInput
        autoCorrect={false}
        autoFocus={false}
        cursorColor="#3CB371"
        handleTextChange={[Function]}
        keyboardType="numeric"
        maxLength={1}
        multiline={false}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        selectionColor="#3CB371"
        style={
          [
            {
              "borderBottomWidth": 4,
              "color": "#000000",
              "fontSize": 22,
              "fontWeight": "500",
              "height": 50,
              "margin": 5,
              "textAlign": "center",
              "width": 50,
            },
            {
              "borderRadius": 10,
              "borderWidth": 4,
              "padding": 10,
            },
            {
              "borderColor": "#DCDCDC",
            },
          ]
        }
        testID="otp_input_1"
        value=""
      />
      <TextInput
        autoCorrect={false}
        autoFocus={false}
        cursorColor="#3CB371"
        handleTextChange={[Function]}
        keyboardType="numeric"
        maxLength={1}
        multiline={false}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        selectionColor="#3CB371"
        style={
          [
            {
              "borderBottomWidth": 4,
              "color": "#000000",
              "fontSize": 22,
              "fontWeight": "500",
              "height": 50,
              "margin": 5,
              "textAlign": "center",
              "width": 50,
            },
            {
              "borderRadius": 10,
              "borderWidth": 4,
              "padding": 10,
            },
            {
              "borderColor": "#DCDCDC",
            },
          ]
        }
        testID="otp_input_2"
        value=""
      />
      <TextInput
        autoCorrect={false}
        autoFocus={false}
        cursorColor="#3CB371"
        handleTextChange={[Function]}
        keyboardType="numeric"
        maxLength={1}
        multiline={false}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        selectionColor="#3CB371"
        style={
          [
            {
              "borderBottomWidth": 4,
              "color": "#000000",
              "fontSize": 22,
              "fontWeight": "500",
              "height": 50,
              "margin": 5,
              "textAlign": "center",
              "width": 50,
            },
            {
              "borderRadius": 10,
              "borderWidth": 4,
              "padding": 10,
            },
            {
              "borderColor": "#DCDCDC",
            },
          ]
        }
        testID="otp_input_3"
        value=""
      />
      <TextInput
        autoCorrect={false}
        autoFocus={false}
        cursorColor="#3CB371"
        handleTextChange={[Function]}
        keyboardType="numeric"
        maxLength={1}
        multiline={false}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        selectionColor="#3CB371"
        style={
          [
            {
              "borderBottomWidth": 4,
              "color": "#000000",
              "fontSize": 22,
              "fontWeight": "500",
              "height": 50,
              "margin": 5,
              "textAlign": "center",
              "width": 50,
            },
            {
              "borderRadius": 10,
              "borderWidth": 4,
              "padding": 10,
            },
            {
              "borderColor": "#DCDCDC",
            },
          ]
        }
        testID="otp_input_4"
        value=""
      />
      <TextInput
        autoCorrect={false}
        autoFocus={false}
        cursorColor="#3CB371"
        handleTextChange={[Function]}
        keyboardType="numeric"
        maxLength={1}
        multiline={false}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        selectionColor="#3CB371"
        style={
          [
            {
              "borderBottomWidth": 4,
              "color": "#000000",
              "fontSize": 22,
              "fontWeight": "500",
              "height": 50,
              "margin": 5,
              "textAlign": "center",
              "width": 50,
            },
            {
              "borderRadius": 10,
              "borderWidth": 4,
              "padding": 10,
            },
            {
              "borderColor": "#DCDCDC",
            },
          ]
        }
        testID="otp_input_5"
        value=""
      />
    </View>
    <View
      style={
        [
          {
            "borderColor": "grey",
            "borderRadius": 10,
            "borderWidth": 1,
            "marginBottom": 10,
            "marginLeft": 10,
            "marginRight": 10,
            "marginTop": 10,
          },
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "borderRadius": 1,
            "opacity": 1,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 16,
                "fontWeight": "400",
              },
            ]
          }
        >
          Submit
        </Text>
      </View>
    </View>
    <View
      style={
        [
          {
            "borderColor": "grey",
            "borderRadius": 10,
            "borderWidth": 1,
            "marginBottom": 10,
            "marginLeft": 10,
            "marginRight": 10,
            "marginTop": 10,
          },
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "borderRadius": 1,
            "opacity": 1,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 16,
                "fontWeight": "400",
              },
            ]
          }
        >
          Clear
        </Text>
      </View>
    </View>
  </View>
</View>
`;
